### PR TITLE
[index] add logo and use it

### PR DESCRIPTION
**Pull request by mineirabrasil**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,10 +9,10 @@
...
             <a href="/">
                 <!--/ this is the root = home-->
-                <img src="Cat.jpg"
-                     alt="Cat">
+                <img src="image/Cat.jpg"
+                     alt="logo Cat">
             </a>
-            <h3>Lorem ipsum dolor sit amet</h3>
+            <h2>Lorem ipsum dolor sit amet</h2>
             <p>other</p>
```
Comment (caloudinou) [inline index.html]: why add p with inner html other ??

**Review by caloudinou — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -9,10 +9,10 @@
...
         </div>
         <nav>
```
Comment (caloudinou) [inline index.html:17]: In nav tag i add list (ul>li>a)

**Review by caloudinou — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (mineirabrasil) [reply index.html:17]: done

Comment (caloudinou) [reply index.html:17]: isn't great correction. modify your menu like this:
`<nav>`
`<ul>`
`<li><a href="/">home</a></li>`
`<li>other</li>`
`</ul>`
`</nav>`

Comment (mineirabrasil) [reply index.html:17]: done with the mentionned corrections
thanks

File: tips_and_tricks/command_LINUX_base.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+LISTER
```
Comment (caloudinou) [inline tips_and_tricks/command_LINUX_base.md]: doesn't add this !! in this commit

File: tips_and_tricks/command_git.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+register commit
```
Comment (caloudinou) [inline tips_and_tricks/command_git.md]: doesn't add this !! in this commit

**Review by caloudinou — CHANGES_REQUESTED** [review]

**Review by caloudinou — APPROVED** [review]

**Pull request merged**